tmva/tmva/inc/TMVA/Envelope.h

```diff
@@ -12,8 +12,9 @@
 #define ROOT_TMVA_Envelope
 
 #include <sstream>
-#include<iostream>
+#include <iostream>
```

Comment:
I don't see why this is required.

Reply:
Yes, you are right!
I forgot to check iostream in header files - only in sources :(